src/config/environnement.ts

```diff
@@ -14,23 +14,12 @@ export const environnement = {
 	api_swagger_path: process.env.API_SWAGGER_PATH || '/documentation',
 	api_swagger_json_path: process.env.API_SWAGGER_JSON_PATH || '/documentation.json',
 	api_swagger_ui_path: process.env.API_SWAGGER_UI_PATH || '/documentation-ui',
-	jwt_secret_name: process.env.JWT_SECRET_KEY_NAME || 'jwtsecret',
+	jwt_secret: process.env.JWT_SECRET || 'jwtsecret',
+	jwt_secret_file: process.env.JWT_SECRET_FILE || '/run/secrets/jwt_secret',
 	jwt_token_expiration_time: process.env.JWT_TOKEN_EXPIRATION_TIME || '2 days',
 	jwt_token_issuer: process.env.JWT_ISSUER_NAME || 'neighbook-api',
 	jwt_token_audience: process.env.JWT_AUDIENCE_NAME || 'neighbook-api',
 	argon2_salt_key_name: process.env.ARGON2_SECRET_KEY_NAME || 'argon2secret',
-	storage_accepted_files_extensions: process.env.STORAGE_ACCEPTED_FILES_EXTENSIONS || 'jpg,jpeg,png,gif',
-	storage_accepted_files_max_size: process.env.STORAGE_ACCEPTED_FILES_MAX_SIZE || 1000000,
-	storage_accepted_files_types: process.env.STORAGE_ACCEPTED_FILES_TYPES || 'image/jpeg,image/png,image/gif',
```

Comment:
We have to keep storage setup like files max size

Reply:
deplacé ligne 43 section storage